rotkehlchen/api/rest.py

```diff
@@ -3748,38 +3748,6 @@ def get_avalanche_transactions(
         result_dict = _wrap_in_result(result, msg)
         return api_response(process_result(result_dict), status_code=status_code)
 
-    def _get_avax_token_info(self, address: ChecksumEvmAddress) -> Dict[str, Any]:
-        avax_manager = self.rotkehlchen.chain_manager.avalanche
-        try:
-            info = avax_manager.get_basic_contract_info(address=address)
-        except BadFunctionCallOutput:
-            return wrap_in_fail_result(
-                f'Address {address} seems to not be a deployed contract',
-                status_code=HTTPStatus.CONFLICT,
-            )
-        return _wrap_in_ok_result(info)
-
-    def get_avax_token_information(
-        self,
-        token_address: ChecksumEvmAddress,
-        async_query: bool,
-    ) -> Response:
-
-        if async_query is True:
-            return self._query_async(command=self._get_avax_token_info, address=token_address)
```

Comment:
Why removing this one?

Reply:
Read the commit or the PR description.

It's not used anywhere. And even if it was we won't have an endpoint like that per chain. We would have one parametrized endpoint.

... but ofc there was a test I missed so I must edit it.